#! /bin/sh /usr/share/dpatch/dpatch-run
## 70-gcc4.6.dpatch by Daniel T Chen <crimsun@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix FTBFS by including cstddef for size_t

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' dansguardian-2.10.1.1~/src/authplugins/ntlm.cpp dansguardian-2.10.1.1/src/authplugins/ntlm.cpp
--- dansguardian-2.10.1.1~/src/authplugins/ntlm.cpp	2008-11-18 06:27:04.000000000 -0500
+++ dansguardian-2.10.1.1/src/authplugins/ntlm.cpp	2011-08-25 16:37:38.000000000 -0400
@@ -27,6 +27,8 @@
 #include "../FDTunnel.hpp"
 #include "../OptionContainer.hpp"
 
+#include <cstddef>
+
 #include <syslog.h>
 
 #include <iconv.h>
